src/app/shared/devices/device-list/device-list.component.test.ts

```diff
@@ -53,9 +70,38 @@ describe('device-list', () => {
         });
     });
 
+    describe('#goToDeviceStatistic', () => {
+        let event;
+        const mqttId = Symbol('device id to show statistic');
+        const statisticLink = 'path to statistic';
+
+        beforeEach(() => {
+            event = {
```

Comment:
Thank you for mention, but it not used in every test, so i'll better leave it in particular tests (beforeEach's).